gef-ui/templates/includes/application-preview/task-comments.njk

```diff
@@ -1,29 +1,31 @@
 {% from "moj/components/banner/macro.njk" import mojBanner %}
 
-<header data-cy="application-preview-page">
+{% if isUkefReviewAvailable or comments.length > 0 %}
+  <header data-cy="application-preview-page">
 
-  {% set bannerHtml %}
+    {% set bannerHtml %}
 
-    {% if isUkefReviewAvailable %}
-      <div class="govuk-!-margin-bottom-6" data-cy="ukef-review">
-        <a href="{{ applicationId }}/review-decision" class="govuk-link">Review UKEF decision</a>
-      </div>
-    {% endif %}
+      {% if isUkefReviewAvailable %}
+        <div class="govuk-!-margin-bottom-6" data-cy="ukef-review">
+          <a href="{{ applicationId }}/review-decision" class="govuk-link">Review UKEF decision</a>
+        </div>
+      {% endif %}
 
-    <div data-cy="latest-comment">
-      {% for comment in comments %}
-        <p><strong>Comments from {{ comment.role }} ({{ comment.userName }})</strong>
-          <br>
-          {{ comment.createdAt | localiseTimestamp('DD MMM YYYY h:mma', timezone) }}</p>
-        <p>{{ comment.comment }}</p>
-      {% endfor %}
-    </div>
+      <div data-cy="latest-comment">
+        {% for comment in comments %}
+          <p><strong>Comments from {{ comment.role }} ({{ comment.userName }})</strong>
+            <br>
+            {{ comment.createdAt | localiseTimestamp('DD MMM YYYY h:mma', timezone) }}</p>
+          <p>{{ comment.comment }}</p>
```

Comment:
what are they allowed to type in the comment box? Is it anything? & do we need to ` | safe` pipe? or do we need to include the `white-space: pre-wrap;` styling?

Reply:
They can type anything, have added Safe filter.
Pre-wrap is already available in the style.